x11-themes/vanilla-dmz-xcursors/vanilla-dmz-xcursors-0.4.5.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="Vanilla-DMZ"
+DESCRIPTION="Style neutral scalable cursor theme (white version)"
+HOMEPAGE="http://jimmac.musichall.cz/"
+SRC_URI="mirror://debian/pool/main/d/dmz-cursor-theme/dmz-cursor-theme_${PV}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE=""
+
+DEPEND="x11-apps/xcursorgen"
+
+S="${WORKDIR}/dmz-cursor-theme-${PV}"
+
+src_compile() {
+	cd "${S}/DMZ-White/pngs" && ./make.sh
+}
+
+src_install() {
+	insinto /usr/share/icons/Vanilla-DMZ/cursors
+	doins "${S}/DMZ-White/xcursors/"* || die "doins failed"
+	insinto /usr/share/icons/Vanilla-DMZ
+	doins "${S}/DMZ-White/index.theme" || die "doins failed"
```

Comment:
How about replacing both `doins` with a single:

    doins -r DMZ-White/{xcursors,index.theme}

Also note that `${S}` is the current directory at this point.

Reply:
Do you mean:
```
insinto /usr/share/icons/Vanilla-DMZ
doins -r DMZ-White/{xcursors,index.theme} || die "doins failed"
```
This would install into `/usr/share/icons/Vanilla-DMZ/xcursors`, but we actually want to install into `/usr/share/icons/Vanilla-DMZ/cursors`.

Reply:
Ah, sorry, I didn't notice the little *x* there.